VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -19,16 +19,23 @@
         "LicenseFilePath": "~/App_Data/VirtoCommerce.lic"
     },
 
-    "EmailSendingOptions": {
-        "SmtpOptions": {
-            "SmtpServer": "",
-            "Port": 587,
-            "Login": "",
-            "Password": ""
+    "Notifications": {
+        "Gateway": "Default",
+        "Default": {
```

Comment:
Better will define two different Options for each sender i.e `SmtpSenderOptions` and `SendGridSenderOptions`  insead of generic `EmailSendingOptions`
and use follow syntax in appsettings.json
```
Notifications : {
Gateway : "Smtp|SendGrid"
.....
}
"SendGrid" : {  ... },
"Smtp" : { ... }
```